http-gateway/config.yaml

```diff
@@ -9,6 +9,10 @@ log:
 apis:
   http:
     address: "0.0.0.0:9100"
+    readTimeout: 10s
+    readHeaderTimeout: 10s
```

Comment:
Here is good description how these timeout works:
https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/

So i suggest to change default behavior to this values:
```
readHeaderTimeout: 4s
readTimeout: 8s
writeTimeout: 16s
idleTimeout: 30s
```